habitipy/cli.py

```diff
@@ -1040,21 +1114,32 @@ def log_op(self, tid):
 
 
 @ToDos.subcommand('add')
-class TodosAdd(ApplicationWithApi):  # pylint: disable=missing-class-docstring
+class TodosAdd(TagsHelper):  # pylint: disable=missing-class-docstring
     DESCRIPTION = _("Add a todo <todo>")  # noqa: Q000
     priority = cli.SwitchAttr(
         ['-p', '--priority'],
         cli.Set('0.1', '1', '1.5', '2'), default='1',
         help=_("Priority (complexity) of a todo"))  # noqa: Q000
+    tags = cli.SwitchAttr(
+        ['-t', '--tags'],
+        default="",
+        help=_("Comma separated list of tags to apply")
+    )
 
     def main(self, *todo: str):
         todo_str = ' '.join(todo)
         if not todo_str:
             self.log.error(_("Empty todo text!"))  # noqa: Q000
             return 1
         super().main()
-        self.api.tasks.user.post(type='todo', text=todo_str, priority=self.priority)
+        response = self.api.tasks.user.post(type='todo', text=todo_str, priority=self.priority)
```

Comment:
This thing returns a Union[Dict,List]. To remove ambiguity, we should just check it is a dict.
```python
if not isinstance(response, dict):
    raise TypeError("API returned a list, but a dict was expected")
```

mypy should see this check and assume response is a dict